kong/plugins/aws-lambda/handler.lua

```diff
@@ -26,11 +27,14 @@ do
   for _, credential_source in ipairs(credential_sources) do
     if credential_source.configured then
       fetch_credentials = credential_source.fetchCredentials
+      fetch_region = credential_source.fetchRegion
       break
     end
   end
 end
 
+local kong_region
```

Comment:
this should probably get a comment that by exception a module level variable is ok